Finjector.Web/ClientApp/src/pages/Paste.tsx

```diff
@@ -65,25 +32,30 @@ const Paste = () => {
       <form>
         <div className="mb-3">
           <p>Paste in a copied account number</p>
-          <textarea
+          <input
             className="form-control"
             id="coa-input"
             value={coa}
             onChange={(e) => setCoa(e.target.value)}
```

Comment:
unrelated to your PR, but should we be doing `.trim()?`

Reply:
probably yes